astropy/io/fits/diff.py

```diff
@@ -249,10 +250,21 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
             are kept in memory or output.  If a negative value is given, then
             numdiffs is treated as unlimited (default: 10).
 
-        tolerance : float, optional
+        rtol : float, optional
             The relative difference to allow when comparing two float values
             either in header values, image arrays, or table columns
-            (default: 0.0).
+            (default: 0.0). Values which satisfy the expression
+
+            .. math::
+
+                \\left| x_1 - x_2 \\right| > \\text{atol} + \\text{rtol} \\cdot \\left| x_1 \\right|
```

Comment:
I think just following the formula from [numpy.allclose](https://docs.scipy.org/doc/numpy-1.10.0/reference/generated/numpy.allclose.html) is less confusing, since the keyword names are the same anyway.